Makefile

```diff
@@ -32,10 +32,12 @@ help:
 #
 #################
 .PHONY: init                                                                            ## Install package dependencies for python
-init: guard-VIRTUAL_ENV $(VIRTUAL_ENV)
-$(VIRTUAL_ENV): $(REQUIREMENTS_FILE)
-	pip install -r $(REQUIREMENTS_FILE)
-	touch $(VIRTUAL_ENV)
+init: guard-VIRTUAL_ENV $(PYTHON_DEPENDENCY_FILE)
```

Comment:
This is a cool dynamic way to install all found `requirements*.txt` files

Reply:
it's clever, yes.  Though I have to ask - should we just have one set of python reqs to support all stacks simultaneously?  I'm not going to block the review on structure but I *am* concerned about all the extra complexity in this project vs other projects.  As far as I'm aware, the only substantial difference that's required is to support launching multiple stacks - is that correct?